jakarta.jdt/pom.xml

```diff
@@ -52,7 +52,7 @@
 						<artifact>
 							<groupId>org.eclipse.jdt.ls</groupId>
 							<artifactId>org.eclipse.jdt.ls.tp</artifactId>
-							<version>${jdt.ls.version}</version>
+							<version>1.7.0.20211216164144</version>
```

Comment:
Why aren't we using the property here, @kathrynkodama?

Reply:
We should be using the property, overlooked this thank you. Updated to utilize the property